### DISCUSSION TITLE
supported bluetooth codecs

### DISCUSSION THREAD

**Discussion by simon-auch** [rern/rAudio#786] [Feature Requests]
I assume this project is using bluez-alsa, if this is correct it would be nice to use a build with a few more enabled audio codecs than the default ([from the bluez-alsa readme](https://github.com/Arkq/bluez-alsa#installation)):
- fdk-aac (when AAC support is enabled with --enable-aac)
- openaptx (when apt-X support is enabled with --enable-aptx and/or --enable-aptx-hd)
- libopenaptx (when apt-X support is enabled and --with-libopenaptx is used)
- libldac (when LDAC support is enabled with --enable-ldac)

I am not entirely sure which of the aptx libraries should be prefered, but either one would be better than nothing.

**Comment by rern** [comment]
`bluez-alsa` on rAudio is based on [bluez-alsa-git](https://aur.archlinux.org/packages/bluez-alsa-git) from AUR. Please try it with your suggested options and share the result.

**Comment by felipegules** [comment]
Hello, 

So I've tried recompiling using the bluez-alsa-git package from AUR with the following on the PKGBUILD **with no success** on having AptX or LDAC Connection. I'm using the bluetooth of the rpi4.
```
  flags=(--with-libopenaptx
         --enable-ldac
         #--enable-mp3lame
         #--enable-mpg123
         #--enable-msbc
         #--enable-ofono
         #--enable-a2dpconf
         #--enable-cli
         #--enable-rfcomm
         #--enable-hcitop
         #--enable-upower
         #--enable-debug
         --enable-aac
         --enable-manpages
         --enable-systemd)
```

**Comment by gorshkovsky** [comment]
hello! I'm trying raudio instead of picore. some things you want to fix. Bluetooth works via AAC and Sbc protocol. How do I get Aptx back?
